docs/src/projects/configuration.md

```diff
@@ -105,7 +105,62 @@ response: [0x1, 0x23, 0x4]
 </details>
 <br>
 
-## Environmental variables
+### Global Configuration
+
+Global configuration file is a [`snfoundry.toml`](https://foundry-rs.github.io/starknet-foundry/appendix/snfoundry-toml.html), 
+which is a common storage for configurations to apply to multiple projects across various directories.
+This file is stored in a predefined location and is used to store profiles that can be used from any location on your computer.
+
+#### Interaction Between Local and Global Profiles
+
+Global config can be overridden by a local config.
+
+If both local and global profiles with the same name are present, local profile will be combined with global profile. That is for any setting defined both in global and local profiles, local setting will be used. For settings not defined in local profile, values from the global profile will be used instead.
```

Comment:
The logic of overriding global default profil should be explicitly mention here, not only in the example below
```suggestion
If both local and global profiles with the same name are present, local profile will be combined with global profile. For any setting defined in both profiles, the local setting will take precedence. For settings not defined in the local profile, values from the corresponding global profile will be used, or if not defined, values from the global default profile will be used instead.
```